test/parallel/test-process-binding-internalbinding-whitelist.js

```diff
@@ -1,21 +1,37 @@
 // Flags: --no-warnings
 'use strict';
 
-require('../common');
+const common = require('../common');
+if (!common.hasCrypto)
+  common.skip('missing crypto');
```

Comment:
This will skip rest of the checks if just `crypto` is not available.

Reply:
We can flip the condition and check if bindings which depend on `crypto` are available, right?